uiplib/scheduler.py

```diff
@@ -11,7 +11,7 @@ class scheduler():
 
     def __init__(self, offline):
         directory = os.path.join(CURR_DIR, PICS_FOLDER)
-        print(time.time())
+        #print(time.time())
```

Comment:
no need to comment. remove it from existence. Terminate it. Let Dogs chew on its bones.